src/components/BreadcrumbsBar/BreadcrumbItem/BreadcrumbItem.tsx

```diff
@@ -8,8 +8,8 @@ import { HideShowEvent } from "../../../constants";
 import { SubIcon, VibeComponentProps } from "../../../types";
 import "./BreadcrumbItem.scss";
 
-const MOUSEENTER = [HideShowEvent.MOUSE_ENTER];
-const MOUSELEAVE = [HideShowEvent.MOUSE_LEAVE];
+const SHOW_TRIGGERS = [HideShowEvent.FOCUS, HideShowEvent.MOUSE_ENTER];
```

Comment:
this is redundant and not intuitive, should be a default for mouse_enter/mouse_leave, mouse_over